resources_pages/install_ds_stack_mac.md

```diff
@@ -585,6 +646,19 @@ PROMPT_COMMAND='history -a; echo'
 # Increase history size
 HISTSIZE=50000
 HISTFILESIZE=50000
+
+# Aliases and functions
+# Note that aliases only are in effect when you are using the shell interactively
+# (e.g. opening the terminal and typing or pasting commands).
+# If you run a bash script or makefile,
+# the original command is used instead of your alias.
+# More verbose file list format
+alias l='ls -vlrthAF'
+# Move, remove, and copy show what was done and prompt before overwriting files.
+alias rm="rm -vI"
+alias mv="mv -vi"
+alias cp="cp -vi"
+alias mkdir="mkdir -vp"
 ```
 
```

Comment:
@joelostblom I was thinking that it would be nice if the terminal could be set up to show info about the current git branch (if it exists), in addition to the conda environment. The solution that I found online is to add the following lines to `~/.bash_profile`, but before the conda initialization block:
```bash
parse_git_branch() {
     git branch 2> /dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/(\1)/'
}
export PS1="\u@\h $(tput bold)\[\e[32m\]\w \[\e[91m\]\$(parse_git_branch)\[\e[00m\]$ "
```
Could you please give it a try and see if it's helpful?

Reply:
Great idea! I taught this  during the course last year, but not too many students added it I think so including it in the setup instructions should be better. There is a built-in solution in git for this[ using the `__git_ps1` variable after sourcing the included git-prompt script](https://github.com/git/git/blob/master/contrib/completion/git-prompt.sh) (I believe this is what is used on git bash for windows by default). I am not sure if I trust them to modify the `PS1` variable though, I feel like it is an easy way to introduce errors... although we could probably just give them an entire line to paste like in your example. The other alternative is a addon package like https://github.com/romkatv/gitstatus#using-from-bash, I think I will try both and see what is the easiest and add the least complexity to the prompt.